sklearn/linear_model/coordinate_descent.py

```diff
@@ -592,7 +603,18 @@ class ElasticNet(LinearModel, RegressorMixin):
         RandomState instance used by `np.random`. Used when ``selection`` ==
         'random'.
 
+    precompute : True | False | array-like
```

Comment:
I'm not sure how we handle cases where new parameters are added. Just a note that until #12805 hasn't happened, changing the order of parameters in the constructor is backward incompatible [sigh].

Reply:
That's too bad. I restore the original order and append ``solver`` at the end...and vote for #12805 :wink: